apps/client/src/checker/components/Question.js

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect } from "react";
```

Comment:
We're going to prevent duplicate code so you can replace the original Question.js